scheduler/src/cook/kubernetes/api.clj

```diff
@@ -890,7 +899,24 @@
     ; init container
     (when use-cook-init?
       (when-let [{:keys [command image]} init-container]
-        (let [container (V1Container.)]
+        (let [container (V1Container.)
+              total-memory-request (+ computed-mem
+                                      (if use-cook-sidecar?
+                                        (get-in sidecar [:resource-requirements :memory-request])
+                                        0))
+              total-memory-limit (+ computed-mem
+                                    (if use-cook-sidecar?
+                                      (get-in sidecar [:resource-requirements :memory-limit])
+                                      0))
+              total-cpu-request (add-as-decimals cpus
+                                                 (if use-cook-sidecar?
+                                                   (get-in sidecar [:resource-requirements :cpu-request])
+                                                   0))
+              total-cpu-limit (add-as-decimals cpus
+                                               (if use-cook-sidecar?
+                                                 (get-in sidecar [:resource-requirements :cpu-limit])
+                                                 0))
```

Comment:
Can we define a function in the `let` for this instead of repeating the code 4 times?